### PR TITLE
fix: dedupe packages

### PR DESCRIPTION
**Pull request by ThaUnknown**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix
[ ] New feature
[x] Other, please explain:

**What changes did you make? (Give an overview)**
Dropped stream-to-blob and stream-to-blob-url, as create-torrent now uses fast-blob-stream which is functionally the same as the 2 [except that fast-blob-stream forces a streamx depencency, while steam-to-blob didn't as it was read only, no write (or other way around, depends on how you look at it)]. Functionally these are the same.

This de-dupes packages. Less depencencies. Smaller bundle size. I don't know why this didn't use file-stream as file and blob are the same thing, except that file has an extra name and mod date.

### CODE REVIEW

**Review by DiegoRBaquero — APPROVED** [review]

**Comment by DiegoRBaquero** [comment]
Why not merge?

**Comment by ThaUnknown** [comment]
> Why not merge?

I feel like I'm not the one to make that call @DiegoRBaquero as I haven't had many interactions with the maintainers so I don't know what the ideology is, or what the target is. If it was up to me I would have probably changed 80% of webtorrent as of now

**Comment by DiegoRBaquero** [comment]
You are a member, as soon as it has approvals you are good to go :)

**Pull request merged**
